docs/userGuide/syntax/code.mbdf

```diff
@@ -125,6 +125,29 @@ To add a heading, add the attribute `heading` with the heading text as the value
 </span>
 </include>
 
+**You can also use multiple features together, as shown below.**
```

Comment:
```suggestion
##### Using multiple features
You can also use multiple features together, as shown below.
```

I feel this will bring more visibility to the section compared to just bolding.